service-loadbalancer/service_loadbalancer.go

```diff
@@ -124,6 +124,10 @@ type service struct {
 	// for this service. For http, it's always :80, for each tcp service it
 	// is the service port of any service matching a name in the tcpServices set.
 	FrontendPort int
+
+	// Host if not empty it will add a new haproxy acl to route traffic using the
```

Comment:
please add a comment here about what gets precedence, I believe the url wins over host header right?